cmd/cloudshell_open/billing.go

```diff
@@ -15,28 +15,22 @@
 package main
 
 import (
-	"bytes"
-	"encoding/json"
+	"context"
 	"fmt"
-	"os/exec"
+
+	"google.golang.org/api/cloudbilling/v1"
 )
 
 // checkBillingEnabled checks if there's a billing account associated to the
-// GCP project ID
+// GCP project ID.
 func checkBillingEnabled(projectID string) (bool, error) {
-	var o bytes.Buffer
-	var e bytes.Buffer
-	cmd := exec.Command("gcloud", "beta", "billing", "projects", "describe", "-q", "--format=json", projectID)
-	cmd.Stdout = &o
-	cmd.Stderr = &e
-	if err := cmd.Run(); err != nil {
-		return false, fmt.Errorf("error determining if billing account is linked: %+v. output=\n%s", err, e.String())
+	client, err := cloudbilling.NewService(context.TODO())
```

Comment:
What is `context.TODO()`?

Reply:
context is a rather complex topic in Go. It’s used for signaling cancellation in this case. 

Here we are not passing a particular context. So in the future we decide to propagate a context, we’d replace these.

Reply:
Ah cool.  Sounds kinda like an alternative to Java's ThreadLocal.  Thanks for the explanation.